plugins/services/src/js/stores/MarathonStore.js

```diff
@@ -471,9 +471,14 @@ class MarathonStore extends GetSetBaseStore {
   injectGroupsWithPackageImages(data) {
     data.items.forEach(item => {
       if (ServiceValidatorUtil.isFrameworkResponse(item)) {
-        item["images"] = FrameworkUtil.extractImageUrls(
-          CosmosPackagesStore.getPackageImages()[item.labels.DCOS_PACKAGE_NAME]
-        );
+        const images = CosmosPackagesStore.getPackageImages()[
+          item.labels.DCOS_PACKAGE_NAME
+        ];
+
+        item["images"] =
+          process.env.NODE_ENV === "production"
+            ? images
+            : FrameworkUtil.extractImageUrls(images);
```

Comment:
The development version filters out any images keys that aren't `["icon-small", "icon-medium", "icon-large"] but in production it does not.

To resolve, you could do one of these:

- move the NODE_ENV condition to the reducer
- change the util method to transform these 3 keys and leave the other keys
- move the variant filter to here

Reply:
Fair point. I think I will go with just fixing the bug as discussed on the latest Sync.

As for the webpack proxy I don't see how this will solve the `https` issue. We run webpack dev server on `http` thus all `https` requests to localhost will fail. I tried `https: true` option then we gonna have this self-signed certificate warning plus some API requests fail with that option being on.